azurerm/resource_arm_data_factory_dataset_sql_server_table_test.go

```diff
@@ -0,0 +1,219 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+)
+
+func TestAccAzureRMDataFactoryDatasetSQLServerTable_basic(t *testing.T) {
```

Comment:
I think this should be:
```suggestion
func TestAccAzureRMDataFactoryDatasetSQLServerTable_update(t *testing.T) {
```

Reply:
Could we create a basic test that is a single step with the bare minimum properties required?